src/instance/structs.jl

```diff
@@ -72,6 +72,8 @@ end
 mutable struct UnitCommitmentInstance
     time::Int
     power_balance_penalty::Vector{Float64}
+    "Penalty for failing to meet reserve requirement."
+    shortfall_penalty::Vector{Float64}
```

Comment:
Just a note that we should come back to this later and document all the remaining fields.